File: DriverMonitor/ClientApp/src/models/DriverActivityFile.ts
export interface DriverActivityFile {
    id?: number;
    driverId: number;
    activityType: string;
    startTime: Date;
    endTime: Date;
    createTime: Date;
}

export interface DriverActivityFileRequest {
    Id?: number;
    DriverId: number;
    ActivityType: string;
    StartTime: Date;
    EndTime: Date;
    CreateTime: Date;
}

File: DriverMonitor/DriverMonitorBackend/Models/DriverActivityFile.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace DriverMonitorBackend.Models
{
    public class DriverActivityFile
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Key, Column(Order = 0)]
        public int Id { get; set; }
        public int DriverId { get; set; }
        public string Activity { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }
        public DateTime CreateTime { get; set; }
    }
}
